mock-data/insertData.js

```diff
@@ -0,0 +1,31 @@
+const db = require('../database/index');
+const mock = require('./generateData');
+
+const userAccounts = mock.generateFirstName(755);
+const ownerAccounts = mock.generateFirstName(100);
+const listingInfo = mock.generateAllListingInfo(100);
+const allBookings = mock.generateAllBookings(100, 755);
+const listingReviews = mock.generateAllReviews(100, 755);
+
+userAccounts.forEach((user) => {
+  db.insertMockData(`INSERT INTO users (userName) VALUES ("${user}")`);
+});
+
+ownerAccounts.forEach((owner) => {
```

Comment:
And actually, there are ways to bulk insert data into mysql without looping, which will be way faster.
for quick reference, see how Ryan did it.